tools/reviewer/webapp/src/app/dashboard/file-changes/services/commit.service.ts

```diff
@@ -0,0 +1,83 @@
+import { Injectable } from '@angular/core';
+
+import { BranchInfo, Thread } from '@/shared/proto';
+import { LocalserverService, NotificationService } from '@/shared/services';
+import { StateService } from './state.service';
+import { ThreadService } from './thread.service';
+
+// Bunch of methods related to commits
+@Injectable()
+export class CommitService {
+  constructor(
+    private localserverService: LocalserverService,
+    private notificationService: NotificationService,
+    private stateService: StateService,
+    private threadService: ThreadService,
+  ) { }
+
+  getCurrentCommitIds(): void {
+    // Commit id before the changed. Left default value.
+    const headCommitId: string = this.stateService.branchInfo.getCommitList()[0].getId();
+    // Commit id of last change with the file. Right default value.
+    const lastCommitId: string = this.stateService.file.getCommitId();
+
+    // Use default value, if custom commit id isn't set
+    this.stateService.leftCommitId = this.stateService.leftCommitId || headCommitId;
+    this.stateService.rightCommitId = this.stateService.rightCommitId || lastCommitId;
+
+    // Get list of threads of current commits state
+    this.stateService.localThreads = this.threadService.getLocalThreads(
+      this.stateService.diff,
+      this.stateService.file,
+      this.stateService.leftCommitId,
+      this.stateService.rightCommitId,
+    );
+  }
+
+  // To don't show UI of changes, if commit don't exist
```

Comment:
s/don't/doesn't

Reply:
Probably,
`To no show UI of changes, if commit doesn't exist`